automatic/lightalloy/update.ps1

```diff
@@ -0,0 +1,27 @@
+import-module au
+
+$releases = 'http://www.light-alloy.ru/download/'
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(?i)(^\s*packageName\s*=\s*)('.*')" = "`$1'$($Latest.PackageName)'"
+      "(?i)(^\s*url32\s*=\s*)('.*')" = "`$1'$($Latest.Url32)'"
+      "(?i)(^\s*checksum32\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+    }
+  }
+}
+
+function global:au_GetLatest {
+  $download_page = Invoke-WebRequest -Uri $releases
+  $url = $download_page.Links | ? href -match "LA_Setup_v[0-9\.]+\.exe$" | % href | select -First 1
+  $version = $url -split 'v' | select -First 1 -Skip 1
+  $version = $version -split '.exe' | select -First 1
+
+  @{
+    Version = $version
+    Url32   = $url32
```

Comment:
@Redsandro Btw this might also be the source of the error message mentioned above